### DISCUSSION TITLE
Create a mask for a InputText field.

### DISCUSSION THREAD

**Discussion by roborobs1023** [AllenDang/giu#125] [Q&A]
How would one go about creating a mask for an InputText field? 
For instance a phone number. Allow it to have mask when blank of ***-***-**** or something similar
I would like to create a textbox with this mask and force that format when a string is entered. I know I can validate the string format with an ozzo-validation statement which I am using.

**Comment by AllenDang** [comment]
It's kind of complicate, text input handling is complex.
